lint/linter.py

```diff
@@ -8,9 +8,8 @@
 import shlex
 import sublime
 
-from . import persist, util
-from .. import highlight_view
-from .const import STATUS_KEY, WARNING, ERROR
+from . import persist, util, ERROR, WARNING
+from .. import highlight_view, status_bar_view
```

Comment:
E.g. here you import stuff from the top level (which is totally ugly BTW but we already have an issue for this open). Now, is this safe? In normal python this is okay.

Reply:
Strictly speaking, this is forbidden. At least, `highlight_view` is probably safe here, the way it is written. So we don'

Reply:
There’s a good chance these aren’t necessary anymore, I’ll try cleaning them away.

Reply:
Nevermind, let's do #899 and then this will clean up one way or the other